shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -223,17 +223,29 @@ public long createForTextureLayer(
           layoutParams.leftMargin = physicalLeft;
           wrapperView.setLayoutParams(layoutParams);
 
-          final View view = platformView.getView();
-          if (view == null) {
+          final View embeddedView = platformView.getView();
+          if (embeddedView == null) {
             throw new IllegalStateException(
                 "PlatformView#getView() returned null, but an Android view reference was expected.");
-          } else if (view.getParent() != null) {
+          } else if (embeddedView.getParent() != null) {
             throw new IllegalStateException(
                 "The Android view returned from PlatformView#getView() was already added to a parent view.");
           }
-          view.setLayoutParams(new FrameLayout.LayoutParams(physicalWidth, physicalHeight));
-          view.setLayoutDirection(request.direction);
-          wrapperView.addView(view);
+          embeddedView.setLayoutParams(new FrameLayout.LayoutParams(physicalWidth, physicalHeight));
+          embeddedView.setLayoutDirection(request.direction);
+
+          // Accessibility is initially disabled, and it's e-enabled by AccessibilityBridge after
```

Comment:
This comment describes the scenario but does not clearly state why this must be set to disabled here specifically at this time.

Reply:
done